EventFilter/CTPPSRawToDigi/python/totemT2Digis_cfi.py

```diff
@@ -3,10 +3,12 @@
 from EventFilter.CTPPSRawToDigi.totemVFATRawToDigi_cfi import totemVFATRawToDigi
 
 totemT2Digis = totemVFATRawToDigi.clone(
-    subSystem = cms.string('TotemT2'),
+    subSystem = 'TotemT2',
     RawToDigi = totemVFATRawToDigi.RawToDigi.clone(
-        testID = cms.uint32(0), #Some ID mismatch in test sample
-        testCRC = cms.uint32(0), # no need to test CRC for diamond frames
-        testECMostFrequent = cms.uint32(0) # show error in the DQM and then DAQ is sending resync, no need to test in the unpacker
+        testID = 0, #Some ID mismatch in test sample
```

Comment:
is it the only clone that needs to be adjusted ?

Reply:
@grzanka Maybe yes, if the variable needs to be declared as a type at least once?

Reply:
Hi again, @grzanka . The TotemTiming was the only one throwing an exception when I first added the tracked T2 parameter, without the present conditional read of the tracked parameter. Looking at the other clones, RP doesn't touch the RawToDigi PSet, and ctppsDiamond accessing it as a dict evidently doesn't overwrite the T2 parameter in the original.